dotCMS/dotsecure/license/server_id.dat

```diff
@@ -0,0 +1 @@
+ba9e1810-8816-434a-bb3a-c880b4581db1
```

Comment:
Exclude this file from the commit please.

Reply:
File has been removed. It was basically automatically added when I build through gradle task and it was skipped during commit